Model/src/BoardPackage/BoardSquare.java

```diff
@@ -1,7 +1,93 @@
 package BoardPackage;
 
-/**
- * Created by unclear on 2/5/18.
- */
+import GamePackage.Player;
+
+import java.util.ArrayList;
+
+ /********************************************************************
+  * The BoardSquare Interface is the base class for all board squares
+  * which will make up the board. It tracks variables universal to
+  * all board squares and has methods for getting and setting those
+  * variables.
+  *
+  * @author David Baas
+  * @version 1.0
```

Comment:
We need to come to an agreement on the Versions. Are we doing numbers, specific date, generic date?

Reply:
I would prefer a version number.